app/src/main/java/com/example/affirmations/MainActivity.kt

```diff
@@ -50,7 +51,9 @@ class MainActivity : ComponentActivity() {
             AffirmationsTheme {
                 // A surface container using the 'background' color from the theme
                 Surface(
-                    modifier = Modifier.fillMaxSize(),
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .statusBarsPadding(),
                     color = MaterialTheme.colorScheme.background
```

Comment:
Nit: We can remove this line, background color should get added by default based on `Themes.xml`